programmers/김서방-찾기.js

```diff
@@ -0,0 +1,10 @@
+function solution(seoul) {
+    var answer = '';
+    
+    for(var i = 0; i<seoul.length; i++){
+        if(seoul[i] == "Kim"){
+            answer = "김서방은 " + i + "에 있다";
```

Comment:
```suggestion
            answer = `김서방은 ${i}에 있다`;
```

`` <- 백틱을 사용하면 "김서방은 " + i + "에 있다" 이 문장을 `김서방은 ${i}에 있다` 이렇게 +를 사용하지 않고
쓸 수 있습니다. ${변수명}을 써주시면 됩니다.